hivemq-edge/src/distribution/conf/examples/configuration/adapters/config-sample-s7.xml

```diff
@@ -47,7 +47,7 @@
                         <includeTimestamp>true</includeTimestamp>
                         <messageHandlingOptions>MQTTMessagePerTag</messageHandlingOptions>
                         <tagName>my-tag-name-1</tagName>
-                        <tagAddress>%I205.0</tagAddress>
+                        <tagDefinition>tag1</tagDefinition>
```

Comment:
Is the tag-name supposed to be part of a URL?
Just asking because we probably need some validation n place to only have characters in the name which work in a URL.

Reply:
we currently have no restraints on it. We convert it to base64 when it is part of an url right now.